### ISSUE TITLE
[v2] Mysql drivers hang when connect over telepresence bridge

### ISSUE THREAD

**Issue by akharlamov** [telepresenceio/telepresence#1967]
## The issue

I use telepresence to connect to a MySQL instance inside Minikube. It works fine until a long enough pause in conversation happens. The next query hangs indefinitely. That results in a Python interpreter being blocked to the point it does not handle signals. Checked with MySQLdb and mysql-python-connector. 

On MySQLd side there are some indications that the protocol is broken.
```
2021-08-06T12:56:12.368270Z 0 [Note] mysqld: ready for connections.
Version: '5.7.34'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
2021-08-06T12:56:55.865491Z 16 [Note] Aborted connection 16 to db: 'airflow' user: 'mysql' host: '172.17.0.1' (Got an error reading communication packets)
2021-08-06T13:00:02.221951Z 587 [Note] Aborted connection 587 to db: 'airflow' user: 'mysql' host: '172.17.0.1' (Got an error reading communication packets)
2021-08-06T13:00:09.513554Z 588 [Note] Aborted connection 588 to db: 'airflow' user: 'mysql' host: '172.17.0.1' (Got an error reading communication packets)
2021-08-06T13:15:20.516072Z 2853 [Note] Aborted connection 2853 to db: 'airflow' user: 'mysql' host: '172.17.0.1' (Got an error reading communication packets)
2021-08-06T13:15:22.666962Z 2904 [Note] Aborted connection 2904 to db: 'airflow' user: 'mysql' host: '172.17.0.1' (Got an error reading communication packets)
```

## Steps to reproduce
Run the test.py script with MySQL 5.7.x container in the cluster.
``` 
import time
import mysql.connector
import random

cnx = mysql.connector.connect(user='mysql', password='mysql',
                              host='mysql.dev2',
                              database='airflow')

for i in range(1, 100):

  print("Starting select")
  cursor = cnx.cursor()

  query = ("SELECT * FROM variable")

  cursor.execute(query)
  for row in cursor:
    print(row)
  cursor.close()

  print("Sleeping for %s seconds", i * 30)
  time.sleep(i * 30)

cnx.close()
```
The script will freeze after 60 seconds delay.

## Hypothesis
MySQL connection is likely to timeout after 60 seconds of silence. However, the information is not propagated back from MySQLd to the client.

**Comment by akharlamov** [comment]
Environment:
* MacOS 11.4
* Minikube v1.22
* Python 3.8.6
* MySQL Connector 2.2.9

**Comment by akharlamov** [comment]
@thallgren Any ideas?

**Comment by thallgren** [comment]
I'm closing this as a duplicate of #1993. The 60 second delay problem will be fixed by PR #1994.
